### PR TITLE
Config changes

### PR DESCRIPTION
**Pull request by Jason2605**
<Please review the guidelines for contributing to this repository. The link is above.>

Resolves #<issue ID here>. <If applicable.>

#### Changes proposed in this pull request: 
- Config Files

### CODE REVIEW

File: Altis_Life.Altis/config/Config_SpawnPoints.hpp [hunk 1]
```diff
@@ -26,56 +26,49 @@ class CfgSpawnPoints {
                 displayName = "Kavala";
                 spawnMarker = "civ_spawn_1";
                 icon = "\a3\ui_f\data\map\MapControl\watertower_ca.paa";
-                licenses[] = { { "rebel", false } };
-                level[] = { "", "", -1 };
+                conditions = "";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_SpawnPoints.hpp]: !Rebel

File: Altis_Life.Altis/config/Config_SpawnPoints.hpp [hunk 2]
```diff
@@ -84,40 +77,35 @@ class CfgSpawnPoints {
...
             };
 
             class Air {
                 displayName = "Air HQ";
                 spawnMarker = "cop_spawn_4";
                 icon = "\a3\ui_f\data\map\Markers\NATO\b_air.paa";
-                licenses[] = { { "cAir", true } };
-                level[] = { "life_coplevel", "SCALAR", 2 };
+                conditions = "FETCH_CONST(life_coplevel) >= 2 && license_cop_cAir";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_SpawnPoints.hpp]: Lazy eval

File: Altis_Life.Altis/config/Config_SpawnPoints.hpp [hunk 3]
```diff
@@ -155,56 +140,49 @@ class CfgSpawnPoints {
                 displayName = "Georgetown";
                 spawnMarker = "civ_spawn_1";
                 icon = "\a3\ui_f\data\map\MapControl\watertower_ca.paa";
-                licenses[] = { { "rebel", false } };
-                level[] = { "", "", -1 };
+                conditions = "";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_SpawnPoints.hpp]: !Rebel

File: Altis_Life.Altis/config/Config_SpawnPoints.hpp [hunk 4]
```diff
@@ -213,40 +191,35 @@ class CfgSpawnPoints {
...
             };
 
             class Air {
                 displayName = "Air HQ";
                 spawnMarker = "cop_spawn_4";
                 icon = "\a3\ui_f\data\map\Markers\NATO\b_air.paa";
-                licenses[] = { { "cAir", true } };
-                level[] = { "life_coplevel", "SCALAR", 2 };
+                conditions = "license_cop_cAir && FETCH_CONST(life_coplevel) >= 2";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_SpawnPoints.hpp]: Lazy eval

File: Altis_Life.Altis/config/Config_Vehicles.hpp [hunk 5]
```diff
@@ -572,7 +572,7 @@ will modify the virtual space and the price of the vehicle, but other informatio
 
     class B_Heli_Light_01_F {
         vItemSpace = 90;
-        licenses[] = { {"pilot"}, {"cAir"}, {"mAir"}, {""} };
+        conditions = "license_civ_pilot || license_cop_cAir || license_med_mAir";
```
Comment (BoGuu) [inline Altis_Life.Altis/config/Config_Vehicles.hpp]: Lazy eval - and for below

**Comment by Jawshy** [comment]
Thanks for submitting a PR, @Jason2605. Could you please describe what you are trying to achieve with these changes?

**Comment by SimZor** [comment]
@Jawshy From what BoGuu and Jason told me they are changing the checks, so it would only call something that returns true or false.

I.e.: ```FETCH_CONST(life_coplevel) > 2``` or ```license_civ_cAir``` or some other function that returns true or false.

**Comment by BoGuu** [comment]
See #128 (config checks). This is a preliminary

**Comment by danielstuart14** [comment]
why this was merged? omg

**Pull request merged**